components/var_dumper/advanced.rst

```diff
@@ -53,16 +53,23 @@ Before calling :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::cloneV
 you can configure these limits:
 
 :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItems`
-    configures the maximum number of items that will be cloned
-    *past the first nesting level*. Items are counted using a breadth-first
+    Configures the maximum number of items that will be cloned
+    *past the minimum nesting depth*. Items are counted using a breadth-first
     algorithm so that lower level items have higher priority than deeply nested
-    items;
+    items. Specifying ``-1`` removes the limit.
```

Comment:
This paragraph still accurately describes behavior of older versions.  For versions < 3.4 we can assume the minimum nesting depth is hard-coded to 1; this is indicated by the next paragraph that says "The default value is 1, which is consistent with older Symfony versions."  The presence of `setMaxItems` is much older than v3.2 - it goes back to Symfony 2 (maybe back to the original introduction of VarCloner).